DOL.WHD.Section14c.Web/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": ["es2015", "dom"],
+    "noImplicitAny": false,
```

Comment:
The squigglies went away. 🤷‍♂️  I did a little more googling and this is basically a compiler warning flag.  Maybe something to consider turning on again in the future to get the full power of type checking, but I think for this push, it's okay to turn off.

Reply:
💯